Source/WebCore/page/EventHandler.cpp

```diff
@@ -4762,7 +4763,8 @@ bool EventHandler::handleTouchEvent(const PlatformTouchEvent& event)
             allTouchReleased = false;
     }
 
-    for (auto& point : points) {
+    for (unsigned index = 0; index < points.size(); index++) {
+        auto& point = points[index];
         PlatformTouchPoint::State pointState = point.state();
         LayoutPoint pagePoint = documentPointForWindowPoint(m_frame, point.pos());
```

Comment:
why is this change needed?

Reply:
Quote from the commit message:
> (WebCore::EventHandler::handleTouchEvent): Change point iteration to
make index value accessible. [...]

The `index` is used down in the `pointerCaptureController.dispatchEventForTouchAtIndex()` call.

Reply:
I see. I missed that, sorry.